framework/validators/UniqueValidator.php

```diff
@@ -69,6 +78,9 @@ public function init()
         if ($this->message === null) {
             $this->message = Yii::t('yii', '{attribute} "{value}" has already been taken.');
         }
+	    if ($this->comboNotUnique === null) {
+		    $this->comboNotUnique = Yii::t('yii', 'Combination [{valueCombo}] of [{attributeCombo}] has already been taken.');
```

Comment:
@Faryshta
> Can you add this to translations messages?

I can't translate this from English into other languages, so I cannot add it.